lambdas/data-migration1/main.tf

```diff
@@ -99,6 +101,8 @@ resource "aws_lambda_function" "data_migration1" {
   environment {
     variables = {
       databaseCredentialSecretArn = var.rds_user_access_secret_arn
+      RulesTable = var.dynamo_tables.rules.name
+      AsyncOperationsTable = var.dynamo_tables.async_operations.name
```

Comment:
This is definitely a nit,  but is there a reason why `RulesTable` is first and not last for alphabetical ordering? Or does it need to be first for a specific tf reason I am unaware of? 🤔

Reply:
Not at all, alpha sorting all the things is an excellent goal!

Reply:
43a43c1c8